.gitignore

```diff
@@ -25,6 +25,7 @@ utils/genspeech/*.zip
 #src/beta-*
 /local/
 coverage/
+src/.vs/
```

Comment:
do we want this?

Reply:
I want it and I suppose that won't disturb anyone